packages/cli/src/commands/run-all-tests/run-all-tests.command.ts

```diff
@@ -67,7 +49,7 @@ const handler: (options: Options) => Promise<void> = async ({
   networkStubbing,
   githubSummary,
   parallelize,
-  parallelTasks,
+  parallelTasks: _parallelTasks,
```

Comment:
Nitpick, convention:

```suggestion
  parallelTasks: parallelTasks_,
```